webapp/components/abstract-test-file-results-table.html

```diff
@@ -8,19 +8,73 @@
   <script>
     // eslint-disable-next-line no-unused-vars
     const AbstractTestFileResultsTable = superClass => class extends superClass {
-      static get is() {
-        return 'abstract-test-file-results-table';
-      }
-
       static get properties() {
         return {
           subtestNames: {
             type: Array,
             value: [],
           },
+          isTestHarness: {
+            type: Boolean,
+            value: false,
+          },
         };
       }
 
+      static get observers() {
+        return ['fetchTestFile(path, testRuns)'];
+      }
+
+      async fetchTestFile(path, testRuns) {
+        if (!path || !testRuns) {
```

Comment:
`this.subtestNames = []; // Clear any existing rows` before this wasn't copied over. Is it no longer possible that `this.subtestNames` should become empty after being non-empty? I haven't read too carefully to discover the invariants, sorry...

Reply:
I think this is no longer needed, but in case I missed something, I added it back, but to a different place: inside the `if` block right before `return`. If the condition doesn't hold, `this.subtestNames` will always be updated at the end of this function.